errors/group/group-deactivated.md

```diff
@@ -0,0 +1,9 @@
+---
+name = "GROUP_DEACTIVATED"
+description = "The group has been deactivated."
```

Comment:
We should add a parameter to the description denoting the deactivation reason

Reply:
@NathanFlurry how was this resolved?

Reply:
Team deactivation isn't actually implemented at the moment, so I'm leaving this as a ticket for a later date. Not essential for MVP.